dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -0,0 +1,343 @@
+					<>
+						<div css={textStyles}>
+							<h1 css={headlineStyles}>
+								{modalType === 'ModalSwitched' &&
+									'We’ve switched you to the new Europe edition of the Guardian'}
+								{modalType === 'ModalDoYouWantToSwitch' &&
+									'Would you like to switch to our new Europe edition?'}
+							</h1>
+							<p css={bodyStyles}>
+								{modalType === 'ModalSwitched' &&
+									'You’re now getting more coverage tailored for readers in Europe'}
+								{modalType === 'ModalDoYouWantToSwitch' &&
+									'We’ve launched a new edition of the Guardian with our global coverage tailored for readers in Europe'}
+							</p>
+							<div css={buttonDivStyles}>
+								{modalType === 'ModalSwitched' && (
+									<Button
+										size={'small'}
+										onClick={() => {
+											confirmNewEdition('EUR');
+										}}
+										cssOverrides={OKButtonStyles}
+									>
+										OK, Thanks
+									</Button>
+								)}
+								{modalType === 'ModalDoYouWantToSwitch' && (
+									<Button
+										size={'small'}
+										onClick={() => dismissModal()}
+										cssOverrides={OKButtonStyles}
+									>
+										{' '}
+										No, Thanks{' '}
+									</Button>
+								)}
+								<Button
+									size={'small'}
+									priority={'subdued'}
+									onClick={() => setSwitchEdition(true)}
+								>
+									{' '}
+									Switch Edition{' '}
+								</Button>
+							</div>
+						</div>
+						<div css={imageStyles}>
+							<SvgFlagsInCircle />
+						</div>
+					</>
+				) : (
+					<div css={editionSectionDivStyles}>
+						<RadioGroup
+							error=""
+							label="Choose your edition"
+							name="colours"
+							orientation="vertical"
+						>
+							<Radio
+								defaultChecked={selectedEdition === 'EUR'}
+								label="Europe edition"
+								value="EUR"
+								onChange={() => setSelectedEdition('EUR')}
+							/>
+							<Radio
+								defaultChecked={selectedEdition === 'UK'}
+								label="UK edition"
+								value="UK"
+								onChange={() => setSelectedEdition('UK')}
+							/>
+							<Radio
+								defaultChecked={selectedEdition === 'US'}
+								label="US edition"
+								value="US"
+								onChange={() => setSelectedEdition('US')}
+							/>
+							<Radio
+								defaultChecked={selectedEdition === 'AU'}
+								label="Australia edition"
+								value="AU"
+								onChange={() => setSelectedEdition('AU')}
+							/>
+							<Radio
+								defaultChecked={selectedEdition === 'INT'}
+								label="International edition"
+								value="INT"
+								onChange={() => setSelectedEdition('INT')}
+							/>
+						</RadioGroup>
+						<Button
+							size={'small'}
+							onClick={() => confirmNewEdition(selectedEdition)}
+						>
+							Confirm
+						</Button>
+					</div>
+				)}
+				{switchEdition && (
+					<Button
+						size={'small'}
+						cssOverrides={closeButtonStyles}
+						onClick={() => dismissModal()}
+					>
+						<SvgClose />
```

Comment:
As this button only contains an SVG we probably need to add a label so that screen readers can announce what this button is doing!